CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Tasks
 * Packages: Dependencies updates ([#2718](https://github.com/dso-toolkit/dso-toolkit/issues/2718))
 
+### Changed
+* Form And Form Group Radios: Update markup ([#2741](https://github.com/dso-toolkit/dso-toolkit/issues/2741))
```

Comment:
Ik heb gezien dat andere breaking changes voorzien zijn van ` **BREAKING**` aan het begin van de changelog-entry

Reply:
volgens mij doen we dat niet meer

Reply:
@tfrijsewijk ?

Reply:
In `dangerfile.ts` checken we er wel op: https://github.com/dso-toolkit/dso-toolkit/blob/master/dangerfile.ts#L171

Maar die check gaat dus niet goed.

Terwijl we op https://www.dso-toolkit.nl/62.30.1/voor-maintainers/change-management-notatie niets over BREAKING in de CHANGELOG reppen.

Ik zou 'm er liever wel in willen. Op termijn zelfs in de commit message. Ik maak een issue aan om hier nog eens naar te kijken.

Voor nu: graag BREAKING toevoegen 🙏

Reply:
Zie ook #2761